coredb-operator/tests/integration_tests.rs

```diff
@@ -40,7 +40,7 @@ mod test {
         let replicas = 1;
 
         // Timeout settings while waiting for an event
-        let timeout_seconds = 30;
+        let timeout_seconds = 60;
```

Comment:
this change is not related to COR-49, it's just to address this comment: https://github.com/CoreDB-io/coredb/pull/24#issuecomment-1362081997

Reply:
also, this is why the operator tests are running in this PR.